web/src/report/components/report-chart/report-chart.component.ts

```diff
@@ -144,6 +233,15 @@ export class ReportChart extends LitElement {
         <google-chart id="membership-distribution"> </google-chart>
         <google-chart id="membership-trends"> </google-chart>
       </div>
+      <div class="select-resource">
```

Comment:
A bit concern with this. I believe this will reduce the size of the chart. The chart should be the same size has all the other charts.

Reply:
it's fine